presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java

```diff
@@ -779,13 +779,15 @@ public void testLeast()
         assertFunction("least(-1.5, -2.3, -5/3)", DOUBLE, -2.3);
         assertFunction("least(1.5, -1.0 / 0.0, 1.0 / 0.0)", DOUBLE, Double.NEGATIVE_INFINITY);
         assertFunction("least(5, 4, CAST(NULL as DOUBLE), 3)", DOUBLE, null);
+        assertFunction("least(5, 4, CAST(NULL as DOUBLE PRECISION), 3)", DOUBLE, null);
```

Comment:
You can remove these two tests you are adding here, and add something else in `TestDoubleOperators.testCastFromVarchar`.